astroquery/eso/core.py

```diff
@@ -606,10 +752,10 @@ def get_headers(self, product_ids, *, cache=True):
                     columns.append(key)
                     column_types.append(type(header[key]))
         # Add all missing elements
-        for i in range(len(result)):
+        for r in result:
```

Comment:
nitpicking, but please try to avoid single letter variables

Reply:
renamed to `item`